tests/migration/online_migration/yast2_migration.pm

```diff
@@ -220,10 +220,13 @@ sub run {
             send_key 'alt-t';
         }
     }
-    assert_screen ['yast2-migration-installupdate', 'yast2-migration-proposal'], 700;
-    if (match_has_tag 'yast2-migration-installupdate') {
+    if (check_screen('yast2-migration-installupdate', 700)) {    # Not all cases have install update message
```

Comment:
Why not continue using the multi-tag?

Reply:
The current situation doesn't fit to use multi-tag any more: the license agreement is shown between yast2-migration-installupdate and yast2-migration-proposal/summary, and the license must be shown, otherwise we should file bug. The flow here is:
installupdate (maybe have it);
license agreement (must have it);
migration proposal (must have it, but it is in the same page with migration_summary, two tags in the same page)